Resources/Prototypes/_RMC14/Entities/Clothing/Hands/marine.yml

```diff
@@ -90,3 +90,10 @@
     sprite: _RMC14/Objects/Clothing/Hands/Gloves/grey.rsi
   - type: Clothing
     sprite: _RMC14/Objects/Clothing/Hands/Gloves/grey.rsi
+
+#TODO RMC14 add fire resistance
+- type: entity
+  parent: CMHandsBlackMarine
+  id: CMHandsBlack
```

Comment:
New content ids should be rmc prefix

I also just dont like this ID in general, maybe do hands black marine fireres since its a child of marine gloves